src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/SPARQLStar2CypherTranslator.java

```diff
@@ -0,0 +1,13 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper;
+
+import se.liu.ida.hefquin.engine.query.impl.TriplePatternImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherQuery;
+
+public interface SPARQLStar2CypherTranslator {
+
+    /**
+     * Translates the given Triple Pattern into a Cypher query, using a given LPG2RDFConfiguration.
+     * If the Triple Pattern is incompatible with the configuration, this method returns null.
```

Comment:
I don't think it's only about being compatible with the configuration. The triple pattern may generally be of a shape for which the (configuration-specific) RDF-star view of the queried LPG is guaranteed to contain no matching triples.